redbeams/src/main/java/com/sequenceiq/redbeams/repository/converter/SslCertificateTypeConverter.java

```diff
@@ -0,0 +1,12 @@
+package com.sequenceiq.redbeams.repository.converter;
+
+import com.sequenceiq.cloudbreak.converter.DefaultEnumConverter;
+import com.sequenceiq.redbeams.api.endpoint.v4.databaseserver.responses.SslCertificateType;
+
+public class SslCertificateTypeConverter extends DefaultEnumConverter<SslCertificateType> {
+
+    @Override
+    public SslCertificateType getDefault() {
+        return SslCertificateType.CLOUD_PROVIDER_OWNED;
```

Comment:
Not sure this is a safe default for unknown DB values. Would not `NONE` be a better choice?